src/main/scala/temple/generate/service/go/GoServiceGenerator.scala

```diff
@@ -55,7 +55,7 @@ object GoServiceGenerator extends ServiceGenerator {
         GoCommonGenerator.generatePackage("comm"),
         GoServiceCommGenerator.generateImports(serviceRoot.module),
         GoServiceCommGenerator.generateStructs(),
-        GoServiceCommGenerator.generateInit(),
+        GoCommonGenerator.generateCommInit(),
```

Comment:
I'm struggling to see how a `CommInit` can be `Common`?

Reply:
```
// Init sets up the Handler object with a list of services from the config
func Init(config *util.Config) *Handler {
	return &Handler{config.Services}
}
```

This is the same regardless of the service, including Auth, so can use the same function to read the same file.

Reply:
I see, that was quite hard to follow from the naming

Reply:
`GoCommon` sounds like it can be used anywhere, whereas I think what you want is `GoCommGenerator`, which then gets extended by `GoServiceCommGenerator` and `GoAuthCommGenerator`

Reply:
See PR description, will address in next PR